dipy/reconst/gqi.py

```diff
@@ -165,31 +169,38 @@ def npa(self, odf, width=5):
 
     Nimmo-Smith et. al  ISMRM 2011
     """
-    #odf = self.odf(s)
+    # odf = self.odf(s)
     t0, t1, t2 = triple_odf_maxima(self.odf_vertices, odf, width)
     psi0 = t0[1] ** 2
     psi1 = t1[1] ** 2
     psi2 = t2[1] ** 2
-    npa = np.sqrt((psi0 - psi1) ** 2 + (psi1 - psi2) ** 2 + (psi2 - psi0) ** 2) / np.sqrt(2 * (psi0 ** 2 + psi1 ** 2 + psi2 ** 2))
-    #print 'tom >>>> ',t0,t1,t2,npa
+    npa = np.sqrt((psi0 - psi1) ** 2 +
+                  (psi1 - psi2) ** 2 +
+                  (psi2 - psi0) ** 2) / \
```

Comment:
yes, please. 

In general, wherever possible, I personally prefer that wrapping to more than one line be done using parentheses, rather than `\`. So I prefer: 

```

    a = (b + c + d + e + 
           f + g)

```

rather than:

```

    a = b + c + d + e + \
           f + g

```